iroh-base/src/ticket/endpoint.rs

```diff
@@ -46,13 +46,13 @@ struct Variant0NodeTicket {
     node: Variant0NodeAddr,
 }
 
-impl Ticket for NodeTicket {
+impl Ticket for EndpointTicket {
     const KIND: &'static str = "node";
```

Comment:
s/node/endpoint? If we want to break all tickets.

Reply:
yeah, I wasn't sure about this one